capi/VideoDecoderCapi.cpp

```diff
@@ -22,135 +22,88 @@
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
-#include "VideoDecoderCapi.h"
+
 #include "VideoDecoderInterface.h"
 #include "VideoDecoderHost.h"
+#include "VideoDecoderCapi.h"
 
-using namespace YamiMediaCodec;
-
-DecodeHandler createDecoder(const char *mimeType)
```

Comment:
Could we change code using "diff friendly" way? Seems you changed all file, but actually only few lines added.